sycl/test-e2e/InvokeSimd/Spec/ImplicitSubgroup/tuple.cpp

```diff
@@ -1,5 +1,6 @@
 // TODO: enable when Jira ticket resolved
 // XFAIL: *
+// XFAIL-TRACKER: https://jira.devtools.intel.com/browse/GSD-4509
```

Comment:
GSD-4509 is never going to be fixed, so these tests are never goign to work. can we just delete all tests failing based of `GSD-4509`? thanks

Reply:
Done.